doc/installation.md

```diff
@@ -0,0 +1,100 @@
+## Introduction
+
+The workflow of the installation is customized via the [control
+file](https://github.com/yast/yast-installation/blob/master/doc/control-file.md), 
+and below we have the principal steps concerning network configuration:
+
+  - Linuxrc
+  - inst_install_inf (What is still needed)
```

Comment:
"What is still needed" sounds like an editorial TODO. IMHO it can be left out of this overview